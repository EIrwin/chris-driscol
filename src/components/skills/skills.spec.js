// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Component from './skills.component';

const fixtures = {
  skills: {
    loves: ['loves1', 'loves2', 'loves3'],
    languages: ['languages1', 'languages2', 'languages3'],
    tools: ['tools1', 'tools2', 'tools3'],
    technologies: ['technologies1', 'technologies2', 'technologies3'],
  },
};

it('renders correctly', () => {
  const tree = renderer.create(<Component {...fixtures} />).toJSON();
  expect(tree).toMatchSnapshot();
});
