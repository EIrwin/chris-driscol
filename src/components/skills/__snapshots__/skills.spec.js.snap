// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="home-section skills"
  id="skills"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12 text-center"
      >
        <h2
          className="section-heading"
        >
          My Skills
        </h2>
        <p
          className="section-subheading text-muted"
        >
          These are some things I've picked up over the years..
        </p>
      </div>
    </div>
    <div
      className="color-gray"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-xs-12 col-md-4"
          >
            <h3
              className="text-center color-silver"
            >
              languages
            </h3>
            <p
              className="text-center color-silver font-size-1_1em"
            >
              languages1, languages2, languages3
            </p>
          </div>
          <div
            className="col-xs-12 col-md-4"
          >
            <h3
              className="text-center color-silver"
            >
              technologies
            </h3>
            <p
              className="text-center color-silver font-size-1_1em"
            >
              technologies1, technologies2, technologies3
            </p>
          </div>
          <div
            className="col-xs-12 col-md-4"
          >
            <h3
              className="text-center color-silver"
            >
              tools
            </h3>
            <p
              className="text-center color-silver font-size-1_1em"
            >
              tools1, tools2, tools3
            </p>
          </div>
        </div>
      </div>
      <div
        className="container ilove"
      >
        I 
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": "pink",
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m20 37.1q-0.6 0-1-0.4l-13.9-13.4q-0.2-0.2-0.6-0.6t-1.3-1.4-1.5-2.2-1.2-2.7-0.5-3.1q0-4.9 2.8-7.7t7.9-2.7q1.4 0 2.8 0.4t2.7 1.3 2.1 1.6 1.7 1.5q0.8-0.8 1.7-1.5t2.1-1.6 2.7-1.3 2.8-0.4q5 0 7.9 2.7t2.8 7.7q0 4.9-5.1 10l-13.9 13.4q-0.4 0.4-1 0.4z"
            />
          </g>
        </svg>
         
        <div
          loop={true}
          strings={
            Array [
              "loves1",
              "loves2",
              "loves3",
            ]
          }
          typeSpeed={100}
        />
      </div>
    </div>
  </div>
</section>
`;
